clients/src/test/java/org/apache/kafka/common/utils/UtilsTest.java

```diff
@@ -448,6 +449,87 @@ public void testMin() {
         assertEquals(1, Utils.min(2, 3, 1));
     }
 
+    @Test
+    public void testMax() {
+        assertEquals(1, Utils.max(1));
+        assertEquals(3, Utils.max(1, 2, 3));
+        assertEquals(3, Utils.max(2, 1, 3, 3));
+        assertEquals(100, Utils.max(0, 2, 2, 100));
+        assertEquals(-1, Utils.max(-1, -2, -2, -10, -100, -1000));
+        assertEquals(0, Utils.max(-1, -2, -2, -10, -150, -1800, 0));
+    }
+
+    @Test
+    public void mkStringTest() {
+        Map<String, String> map = new LinkedHashMap<>();
+        map.put("key1", "val1");
+        map.put("key2", "val2");
+        map.put("key3", "val3");
+        String result = Utils.mkString(map, "__begin__", "__end__", "=", ",");
+        assertEquals("__begin__key1=val1,key2=val2,key3=val3__end__", result);
+
+        String result2 = Utils.mkString(new HashMap<>(), "__begin__", "__end__", "=", ",");
+        assertEquals("__begin____end__", result2);
+
+        String result3 = Utils.mkString(null, "__begin__", "__end__", "=", ",");
+        assertEquals("__begin____end__", result3);
+
+        map.clear();
+        map.put("key4", "val4");
+        map.put("key5", "val5");
+        String result4 = Utils.mkString(map, null, null, " : ", ";");
+        assertEquals("key4 : val4;key5 : val5", result4);
+
+        map.clear();
+        map.put("key6", "val6");
+        String result5 = Utils.mkString(map, null, null, " : ", ";");
+        assertEquals("key6 : val6", result5);
+    }
+
+    @Test
+    public void parseMapTest() {
+        Map<String, String> map1 = Utils.parseMap("k1=v1,k2=v2,k3=v3", "=", ",");
+        assertEquals(3, map1.size());
+        assertTrue(map1.containsKey("k1"));
+        assertTrue(map1.containsKey("k2"));
+        assertTrue(map1.containsKey("k3"));
+        assertTrue(map1.containsValue("v1"));
+        assertTrue(map1.containsValue("v2"));
+        assertTrue(map1.containsValue("v3"));
+
+        Map<String, String> map2 = Utils.parseMap(null, "=", ",");
+        assertEquals(0, map2.size());
+
+        Map<String, String> map3 = Utils.parseMap("k4=v4,k5=v5=vv5=vvv5", "=", ",");
+        assertEquals(2, map3.size());
+        assertTrue(map3.containsKey("k4"));
+        assertTrue(map3.containsKey("k5"));
+        assertTrue(map3.containsValue("v4"));
+        assertTrue(map3.containsValue("v5=vv5=vvv5"));
```

Comment:
I think to test `k5` key get value is equals to`v5=vv5=vvv5` is more better, WDTY